www/package.json

```diff
@@ -39,28 +42,32 @@
     "@testing-library/react": "13.4.0",
     "@types/styled-components": "5.1.26",
     "apollo-absinthe-upload-link": "1.7.0",
-    "buffer": "6.0.3",
+    "babel-polyfill": "^6.26.0",
+    "browserify-zlib": "^0.2.0",
+    "buffer": "^6.0.3",
     "country-code-lookup": "0.0.20",
     "cra-template": "1.2.0",
     "d3-hierarchy": "3.1.2",
     "d3-selection": "3.0.0",
     "emoji-mart": "3.0.1",
+    "events": "^3.3.0",
     "filesize": "8.0.7",
     "fireworks-js": "1.4.1",
     "forge-core": "1.4.6",
     "fuse.js": "6.6.2",
     "grommet": "2.27.0",
     "history": "5.3.0",
-    "honorable": "0.144.0",
+    "honorable": "0.148.0",
     "honorable-recipe-mp": "0.3.0",
-    "honorable-theme-default": "0.65.0",
+    "honorable-theme-default": "0.66.0",
     "js-yaml": "4.1.0",
     "lodash": "4.17.21",
     "memoize-one": "6.0.0",
     "moment": "2.29.4",
     "phoenix": "1.6.15",
     "pluralsh-absinthe-socket-apollo-link": "0.2.0",
-    "pluralsh-design-system": "1.241.0",
+    "pluralsh-design-system": "1.288.0",
+    "process": "^0.11.10",
```

Comment:
Can you change to exact versions here and for other packages?

Reply:
Done by @floreks